### PR TITLE
Add task for Pluto Tool

### PR DESCRIPTION
**Pull request by urvashigupta7**
This task makes it possible to use Pluto within Tekton pipelines.

- [x] Follows the [authoring recommendations](https://github.com/tektoncd/catalog/blob/main/recommendations.md)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality of task changed or new task added)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [x] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [x] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [x] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [x] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [x] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [x] mandatory `spec.description` follows the convention

### CODE REVIEW

**Comment by urvashigupta7** [comment]
@vinamra28 Can you please enable tests here? Thanks :)

**Review by vinamra28 — COMMENTED** [review]
/ok-to-test

File: task/pluto/0.1/pluto.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: pluto
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: Deprecation, Kubernetes, APIVersions
+    tekton.dev/displayName: "Pluto"
```
Comment (vinamra28) [inline task/pluto/0.1/pluto.yaml:10]: can we have one more annotation over here
`tekton.dev/categories: category name`. You can find the list of categories over [here](https://github.com/tektoncd/hub/pull/283/files)

**Review by vinamra28 — COMMENTED** [review]
just one comment needs to be addressed otherwise PR looks good

**Comment by vinamra28** [comment]
@urvashigupta7 are you still working on this? If not then shall I close it?

**Pull request closed without merging**